<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(16,30,41,240);
border-radius:10px;

</string>
  </property>
  <widget class="QPushButton" name="hayatkonu1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>360</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#hayatkonu1{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#hayatkonu1:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,255);
background-position:clac(100%-10px)center;
}
QPushButton#hayatkonu1{
background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px
 }
</string>
   </property>
   <property name="text">
    <string>S A G L I K L I  H A Y A T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hayatkonu2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>360</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#hayatkonu2{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#hayatkonu2:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,255);
background-position:clac(100%-10px)center;
}
QPushButton#hayatkonu2{
background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px
 }
</string>
   </property>
   <property name="text">
    <string>G Ü V E N L İ  H A Y A T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hayatkonu3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>360</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#hayatkonu3{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#hayatkonu3:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,255);
background-position:clac(100%-10px)center;
}
QPushButton#hayatkonu3{
background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px
 }
</string>
   </property>
   <property name="text">
    <string>Ü L K E M İ Z D E  H A Y A T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hayatkonu4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>390</y>
     <width>360</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#hayatkonu4{
background-color:rgba(2,65,118,255);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#hayatkonu4:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2,65,118,255);
background-position:clac(100%-10px)center;
}
QPushButton#hayatkonu4{
background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgb(255,255,255);
padding-bottom:7px
 }
</string>
   </property>
   <property name="text">
    <string>D O Ğ A D A  H A Y A T</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
